x-pack/plugins/canvas/public/render_functions/visualize.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getVisualizeLoader } from 'ui/visualize/loader';
```

Comment:
@ppisljar or @w33ble - did you have to do anything special to import this from `ui/public`?  I seem to get errors when I try, though the code has probably changed a lot.  Like, my renderer is inside `canvas_plugin_src/renderers`

<img width="530" alt="screen shot 2018-11-08 at 11 20 08 am" src="https://user-images.githubusercontent.com/16563603/48211807-46379b80-e348-11e8-8bff-7633f31716d2.png">

any ideas?

Reply:
nm I think this is why you added the client side render_functions.  I will follow suit with my exploration.